code/game/machinery/doors/airlock.dm

```diff
@@ -149,6 +149,12 @@ About the new airlock wires panel:
 		damage_deflection = AIRLOCK_DAMAGE_DEFLECTION_R
 	update_icon()
 
+/obj/machinery/door/airlock/on_construction()
+	// Remove shielding to prevent metal/plasteel duplication
+	security_level = AIRLOCK_SECURITY_NONE
+	modify_max_integrity(normal_integrity)
+	damage_deflection = AIRLOCK_DAMAGE_DEFLECTION_N
```

Comment:
```suggestion
/obj/machinery/door/airlock/on_construction()
	remove_shielding()
	. = ..()

// Remove shielding to prevent metal/plasteel duplication
/obj/machinery/door/airlock/proc/remove_shielding()
	security_level = AIRLOCK_SECURITY_NONE
	modify_max_integrity(normal_integrity)
	damage_deflection = AIRLOCK_DAMAGE_DEFLECTION_N
```